packages/flutter_tools/lib/src/base/command_help.dart

```diff
@@ -0,0 +1,71 @@
+import 'dart:math' show max;
+
+import '../globals.dart';
+import 'io.dart' as io;
```

Comment:
never import dart:io in the tool, there's a shim we use instead

Reply:
I cannot find where I import `dart:io`, I thought that `io.dart` is the shim  🤔

Reply:
Yeah sorry I misread 'io.dart' as 'dart:io' :-)